backend/src/Installer.ts

```diff
@@ -1,35 +1,30 @@
-import { CLI_DOWNLOAD } from './constants'
 import debug from 'debug'
 import cli from './cliInterface'
 import preferences from './preferences'
-import binaryInstaller from './binaryInstaller'
 import semverCompare from 'semver/functions/compare'
 import environment from './environment'
-import EventBus from './EventBus'
 import Logger from './Logger'
-import https from 'https'
-import fs from 'fs'
 import path from 'path'
 import { existsSync } from 'fs'
+import binaryInstaller from './binaryInstaller'
+import EventBus from './EventBus'
 
 const d = debug('installer')
 
 export type ProgressCallback = (percent: number) => void
 
 interface InstallerArgs {
-  name: string
```

Comment:
Please continue to use this.name and this.version instead of using the resourced definitions. I think we might be able to remove the resources definitions altogether.